tests/interop/global-cache-test.js

```diff
@@ -0,0 +1,194 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import { run } from '@ember/runloop';
+import DefaultSchema from 'ember-m3/services/m3-schema';
+import sinon from 'sinon';
+import { CUSTOM_MODEL_CLASS } from 'ember-m3/-infra/features';
+import { HAS_MODEL_PACKAGE } from 'ember-m3/-infra/packages';
+import require from 'require';
+
+let Model, attr;
+if (HAS_MODEL_PACKAGE) {
```

Comment:
Since this test can run with different `Model` and `attr` implementations, do the tests run or need to run multiple combinations/permutations?

Reply:
They don't. `Model` and `attr` will always be the same, only their source module will change.